collectors/ebpf.plugin/ebpf.c

```diff
@@ -302,19 +302,23 @@ void write_io_chart(char *chart, char *family, char *dwrite, char *dread, netdat
  *
  * @param type    the chart type
  * @param id      the chart id
+ * @param title   the chart title
  * @param units   the units label
  * @param family  the group name used to attach the chart on dashaboard
  * @param order   the chart order
  */
 void ebpf_write_chart_cmd(char *type
     , char *id
+    , char *title
     , char *units
     , char *family
     , int order)
 {
-    printf("CHART %s.%s '' 'title' '%s' '%s' '' line %d %d\n",
+    (void)title;
```

Comment:
Did you forget to delete it?

Reply:
You are right, I missed to remove, thank you!